lib/public/components/common/table/rows.js

```diff
@@ -32,6 +32,8 @@ const cell = (rowId, column, rowData) => {
     // Flag defining if the text must have a popover displayed at hover containing the full text
     const isBalloon = typeof balloon === 'function' ? balloon(cellData) : Boolean(balloon);
 
+    const errorStatus = rowData.status === 'ERROR';
```

Comment:
Now that I see it, I think that this logic `(if rowData === 'ERROR') // add class danger` would better fit in the column configuration, not directly in the table

Reply:
Tried to accomplish this in ```environmentOverviewComponent.js``` but I can't get it to work. Further assistance needed.